packages/ember-glimmer/tests/utils/global-helpers-hash.js

```diff
@@ -0,0 +1 @@
+../../../ember-htmlbars/lib/helpers.js
```

Comment:
This is slightly confusing to me. Why would ember-glimmer point at ember-htmlbars helpers?

Can you help me understand?